cmd/ocm/cluster/create/create.go

```diff
@@ -164,8 +201,13 @@ func run(cmd *cobra.Command, argv []string) error {
 		Version(
 			cmv1.NewVersion().
 				ID(clusterVersion),
-		).
-		Build()
+		)
+	if args.expiration != nil {
+		clusterBuild = clusterBuild.ExpirationTimestamp(
+			args.expiration.Time,
```

Comment:
Something seems to be wrong here (perhaps a bug in the CS API, but it could be in the spf13/flag module):
```
$ ocm cluster create --expiration=7d mycluster
Error: invalid argument "7d" for "--expiration" flag: time: unknown unit d in duration 7d
```

Reply:
Tracked this down, and it's actually `time.Duration` not supporting days. I don't quite feel like writing a custom parser for it, so I've updated the help text to suggest using `72h` instead of `3d`.